src/swap/changelly.js

```diff
@@ -143,8 +143,9 @@ export function makeChangellyPlugin(
       'Content-Type': 'application/json',
       'api-key': apiKey,
       sign,
-      'X-Promo-Code': promoCode
+      'X-Promo-Code': ''
```

Comment:
Delete this line. No need to send the header at all unless we have the code. That was the point of the `if`